kano_init_flow/stages/desktop/main.py

```diff
@@ -397,12 +405,20 @@ def _setup_fourth_scene(self):
 
         return scene
 
+    def _desktop_icon_enter_cb(self, widget, event, user=None):
+        if widget.get_opacity() < 1.0:
+            widget.set_opacity(0.75)
+
+    def _desktop_icon_leave_cb(self, widget, event, user=None):
+        if widget.get_opacity() < 1.0:
+            widget.set_opacity(0.5)
```

Comment:
Is this supposed to match the value set on line 368? If so would you mind storing it as a "constant" at the top to make it obvious that they are to be the same value?